src/main/java/com/michalsydoryk/app/coordinates/Coordinates2D.java

```diff
@@ -0,0 +1,43 @@
+package com.michalsydoryk.app.coordinates;
+
+import java.util.Comparator;
+import java.util.Objects;
+
+public class Coordinates2D {
+    private final int x;
```

Comment:
Public? One-class package? What about encapsulation?

Reply:
Coordinates are used by Board2D and Game classes which are in 2 different packages, so I will merge this packages or leave it as it is.